shared_infra/dynamo_write_heartbeat/src/test_dynamo_write_heartbeat.py

```diff
@@ -0,0 +1,50 @@
+# -*- encoding: utf-8 -*-
+
+import pytest
+import boto3
+import os
+import dynamo_write_heartbeat
+
+TABLE_NAMES = 'Test1, Test2'
+ENDPOINT_URL = 'http://localhost:8000'
+
+
+@pytest.yield_fixture(autouse=True)
+def run_around_tests():
+    dynamodb = boto3.client('dynamodb', endpoint_url=ENDPOINT_URL)
+    table_names = [t.strip(' ') for t in TABLE_NAMES.split(",")]
```

Comment:
This is repeating the logic in the code under test. I think it would be preferable to do something like:

```
TABLE_NAMES = ['Test1', 'Test2']
TABLE_NAME_STR = ','.join(TABLE_NAMES)
```